app/src/main/java/org/gdg/frisbee/android/activity/PulseActivity.java

```diff
@@ -89,30 +84,30 @@ public void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setHomeButtonEnabled(true);
 
         mFetchGlobalPulseTask = mClient.getPulse(new Response.Listener<Pulse>() {
-                @Override
-                public void onResponse(final Pulse pulse) {
-                    getSupportActionBar().setListNavigationCallbacks(mSpinnerAdapter, PulseActivity.this);
-                    App.getInstance().getModelCache().putAsync("pulse_global", pulse, DateTime.now().plusDays(1), new ModelCache.CachePutListener() {
-                         @Override
-                         public void onPutIntoCache() {
-                            mPulseTargets.addAll(pulse.keySet());
-                            initSpinner();
-                         }
-                    });
-                }
-            }, new Response.ErrorListener() {
-                @Override
-                public void onErrorResponse(VolleyError volleyError) {
-                    Crouton.makeText(PulseActivity.this, getString(R.string.fetch_chapters_failed), Style.ALERT).show();
-                    Timber.e("Could'nt fetch chapter list", volleyError);
-                }
-            }
+                                                     @Override

```

Comment:
Good point. Maybe it's some problem with my settings.
Today I noticed that I had 4 space indentation on the XML files and I should have only two.
I'm using Novoda's settings, which were wrong so I immediately did a PR to fix that XML settings.